packages/rosaenlg-lib/test/pug/fr_FR/sentence_negative.pug

```diff
@@ -202,6 +208,65 @@ mixin SUJET_MP_ref(obj, params)
     },
   };
 
+//-  Le tyran n'a vraiment fourni aucune aide au jeune aventurier.
+
+
+- const SUJET_TYRAN = {};
+mixin SUJET_TYRAN_ref(obj, params)
```

Comment:
It is a good example. Can you add it in `sentences_french.adoc`?

Reply:
added a similar one.